File: shoring/flywheel/src/test/java/com/panopset/tests/flywheel/VariableNamedFilesTest.java
package com.panopset.tests.flywheel;

import java.io.IOException;
import org.junit.jupiter.api.Test;

public final class VariableNamedFilesTest {

  /**
   * Test variable named template. See <b>variableNamedTemplateTest.txt</b>.
   */
  @Test
  void testVariableNamedTemplate() throws IOException {
    new SimpleTest().comparisonTest("variableNamedTemplateTest.txt", "variableNamedTemplateResult.txt",
        "variableNamedTemplateExpected.txt");
  }

  /**
   * Test variable named list. See <b>variableNamedListTest.txt</b>
   */
  @Test
  void testVariableNamedList() throws IOException {
    new SimpleTest().comparisonTest("variableNamedListTest.txt", "variableNamedListResult.txt",
        "variableNamedListExpected.txt");
  }

}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/flywheel/VariableNamedFilesTest.kt
package com.panopset.tests.flywheel

import org.junit.jupiter.api.Test

class VariableNamedFilesTest {
    /**
     * Test variable named template. See **variableNamedTemplateTest.txt**.
     */
    @Test
    fun testVariableNamedTemplate() {
        SimpleTest().comparisonTest(
            "variableNamedTemplateTest.txt", "variableNamedTemplateResult.txt",
            "variableNamedTemplateExpected.txt"
        )
    }

    /**
     * Test variable named list. See **variableNamedListTest.txt**
     */
    @Test
    fun testVariableNamedList() {
        SimpleTest().comparisonTest(
            "variableNamedListTest.txt", "variableNamedListResult.txt",
            "variableNamedListExpected.txt"
        )
    }
}
